mlflow/tracing/utils.py

```diff
@@ -101,3 +109,53 @@ def build_otel_context(trace_id: int, span_id: int) -> trace_api.SpanContext:
         # yet so always set it to False.
         is_remote=False,
     )
+
+
+def traces_to_df(traces: List["mlflow.entities.Trace"]) -> "pandas.DataFrame":
+    """
+    Convert a list of MLflow Traces to a pandas DataFrame with one column called "traces"
+    containing string representations of each Trace.
+    """
+    import pandas as pd
+
+    rows = [
+        _TraceRow(
+            request_id=trace.info.request_id,
+            timestamp_ms=trace.info.timestamp_ms,
+            status=trace.info.status,
+            execution_time_ms=trace.info.execution_time_ms,
+            request=trace.data.request,
+            request_metadata=trace.info.request_metadata,
+            spans=trace.data.spans,
+            tags=trace.info.tags,
+            response=trace.data.response,
+        )
+        for trace in traces
+    ]
+    return pd.DataFrame.from_records([row.to_dict() for row in rows])
+
+
+@dataclass
+class _TraceRow:
```

Comment:
Do we need this data class? It seems the dataclass is constructed and converted to dictionary immediately. Would it be simpler to define `to_pandas_dataframe_row()` method in the `Trace` dataclass?